AmplifyPlugins/Core/AWSPluginsCore/Model/GraphQLRequest/GraphQLRequest+AnyModelWithSync.swift

```diff
@@ -0,0 +1,98 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import Foundation
+
+public typealias SyncQueryResult = PaginatedList<AnyModel>
+public typealias MutationSyncResult = MutationSync<AnyModel>
+
+/// Extension methods that are useful for `DataStore`. The methods consist of conflict resolution related fields such
```

Comment:
> Extension methods that are useful for `DataStore`

Wouldn't these belong in DataStore category then, or do you use them in API category as well?

Reply:
currently, API Category has a set of integration tests which uses theses against a conflict resolution enabled backend and so they will also need to move over to DataStore. I think they can go either way (thoughts around decorators being public facing or only extensions should be public), let me know what you think